pyproject.toml

```diff
@@ -147,3 +147,6 @@ requires = [
     "cython >= 3.0",
 ]
 build-backend = "hatchling.build"
+
+[project.scripts]
+biotite-setup-ccd = "biotite.setup_ccd:main"
```

Comment:
Since this script is rather run very rarely, I think we do not need a dedicated command. Running
```
python -m biotite.setup_ccd
```
would be sufficient in my opinion.

Reply:
makes sense! removed